source/extensions/filters/common/ext_authz/ext_authz_grpc_impl.cc

```diff
@@ -43,9 +43,7 @@ void GrpcClientImpl::check(RequestCallbacks& callbacks,
 
 void GrpcClientImpl::onSuccess(std::unique_ptr<envoy::service::auth::v2::CheckResponse>&& response,
                                Tracing::Span& span) {
-  ASSERT(response->status().code() != Grpc::Status::GrpcStatus::Unknown);
```

Comment:
Can you potentially add a test for this?

Reply:
Good call, I will do.